File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.py
def solution(arr1, arr2):
    answer = []
    
    for i in range(len(arr1)):
        row = []
        for j in range(len(arr2[0])):
            s = 0
            for k in range(len(arr1[0])):
                s += arr1[i][k] * arr2[k][j]
            row.append(s)
        answer.append(row)
    
    return answer

File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.js
function solution(arr1, arr2) {
    const answer = arr1.map(() => arr2[0].map(() => 0));
    
    for (let i = 0; i < arr1.length; i++) {
        for (let j = 0; j < arr2[0].length; j++) {
            for (let k = 0; k < arr1[0].length; k++) {
                answer[i][j] += arr1[i][k] * arr2[k][j];
            }
        }
    }
    
    return answer;
}
